src/main/java/com/networknt/schema/RefValidator.java

```diff
@@ -66,16 +63,47 @@ public RefValidator(String schemaPath, JsonNode schemaNode, JsonSchema parentSch
                 schema = parentSchema.findAncestor();
             } else {
                 refValue = refValue.substring(index);
-                if (refValue.equals("#")) {
-                    schema = parentSchema.findAncestor();
-                } else {
-                    JsonNode node = parentSchema.getRefSchemaNode(refValue);
-                    if (node != null) {
-                        schema = new JsonSchema(mapper, refValue, node, parentSchema);
-                    }
-                }
             }
         }
+        if (refValue.equals(REF_CURRENT)) {
+            schema = parentSchema.findAncestor();
+        } else {
+            JsonNode node = parentSchema.getRefSchemaNode(refValue);
+            if (node != null) {
+                schema = new JsonSchema(mapper, refValue, node, parentSchema);
+            }
+        }
+    }
+    
+    private boolean isRelativePath(String schemaUrl) {
+    	return !schemaUrl.startsWith("http");

```

Comment:
What about relative **local** paths? With this change I can't load local schemas anymore using `$ref`.